deploy/resources/addons/dashboard/v0.1.0/gcr-tekton-dashboard.yaml

```diff
@@ -0,0 +1,215 @@
+# ------------------- Extension Resource Definition ------------------- #
```

Comment:
should this be removed?

Reply:
v0.1.0

Reply:
Yes, the v0.1.0 is not necessary

Reply:
I remove webhooks-extension_release.yaml from v0.1.1.  The webhook-extension should be a separate addon.